doc/source/building_containers.rst

```diff
@@ -0,0 +1,130 @@
+Building Container Images
+=========================
+
+.. hint:: **Abstract**
+
+   This page describes how to build container images with KIWI.
+   This description applies for version |version|.
+
+Currently KIWI supports OCI and Docker container formats. Container
+images work the same way as any other image type in KIWI, the
+:ref:`prepare step <prepare-step>` creates the build root and configures the
+system and the :ref:`create step <create-step>` configures and creates
+the resulting image with required container format. KIWI makes use of
+:command:`umoci` and :command:`skopeo` tools in order to operate with
+container images, because of that those tools must be present in
+the build environment, KIWI has a runtime check to validate skopeo and
+umoci are present in ``PATH`` when building a container image. Building
+container images in KIWI is like building any other kind of image, the
+specific part in the description file is included under the ``<type>``
+section. Contcretely, it hanldes the following OCI or Docker like
```

Comment:
Typo: Concretly
Or use "In practice", "Precisely"... depending what you like more.